index.html

```diff
@@ -0,0 +1,292 @@
+                
+                <li id="noMenu"><i class="fa fa-times" onclick="closeMenu()"></i></li>
+              
+                <li><a href="#tonicx" onclick="closeMenu()" class="noMenu1">Portfolio </a></li>
+                <li><a href="#abouts" onclick="closeMenu()" class="noMenu1">About </a></li>
+                <li><a href="#contacts" onclick="closeMenu()" class="noMenu1">Contact </a></li></ul>
+
+         
+           
+            
+          </nav>
+        </header>
+        <div class="details">
+          <h1>I'm Kaweesi Glad to see you!</h1>
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <a id="connect" href="index.html">LET'S CONNECT</a>
+          <ul class="details-list1">
+            <li>
+              <a href=""> <img src="images/twitter.png" alt="Twitter" /></a>
+            </li>
+            <li>
+              <a href=""> <img src="images/Linkedin.png" alt="LinkedIn" /></a>
+            </li>
+            <li>
+              <a href=""> <img src="images/medium.png" alt="Media" /></a>
+            </li>
+            <li>
+              <a href=""> <img src="images/Vector-1.png" alt="image" /></a>
+            </li>
+            <li>
+              <a href=""> <img src="images/Vector-3.png" alt="image" /></a>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div class="wrapper2">
+        
+        
+
+        <div id="portfolio" class="works"></div>
+
+        <!-- desktop popup -->
+        <div id="sectionWrap" class="modal">
+          <section class="desktop">
+           <div class="desktopIntro"> <h2 class="deskHeading"></h2>
+            <ul class="deskButtons">
+              <li class="deskList1"  ></li>
+              <li class="deskList2"></li>
+              <li  class="deskList3"></li>
+            </ul>
+            <div class="closeProj"><i class="fa fa-times" onclick="closeall()"></i></div>
+           </div>
+            <div><img src="/images/snap3.png" class="deskImag" alt="project-image"></div>          
+            <div class="desktopFlex">
+            <p class="deskpara"></p>
+            <aside>
+            <ul class="deskButtonlist">
+                           
+            </ul>
+            <div class="liner"></div>
+            <div class="separater">
+            <div class="seeLive">
+           <a href="" id="liveLink"> see Live  <img src="images/live.png" alt="github pages"> </a></div>
+           <div class="seeSource">
+            <a href="" id="sourceLink"> See Source <img src="images/vect1.png" alt="github"> </a></div></div>
+          </aside></div>
+          </section>
+        </div>
+
+
+
+  <!-- mobile popup -->
+  <div id="mobileWrap" class="modal">
```

Comment:
- [ ] Good job so far, good project implementation :+1: But when we click on the project button on the desktop version,  the popup is showing up however there are a few differences between your design and your chosen Figma design, especially the height and width and also some elements. Please make sure that your design sticks to the Figma design as much as possible to give a good user experience and have a wonderful website working correctly :smile: 

- [ ] Your popup on the mobile version does not adapt correctly. I think this gives a slightly unpleasant look which affects your design. Please make sure that the popup adapts well to the mobile and even desktop versions and that it has the dimensions, size, and elements that make it similar to your Figma template. Again please try to make your design match with the Figma template because it is really important for the user experience, also important to pay attention to detail :+1: 

- Please check this recorded video to see what my comments are talking about more easily :arrow_down: 

https://www.loom.com/share/18a82ea1a33b4ff9a41c563783d0496b

- Check the difference between your design and Figma template :point_down: 

#### Mobile version
![Screenshot from 2022-09-08 13-47-20](https://user-images.githubusercontent.com/6615394/189139412-eab06869-4197-444e-b085-d50dfeee199d.png)

#### Desktop version
![Screenshot from 2022-09-08 13-46-32](https://user-images.githubusercontent.com/6615394/189139423-5b8720d1-6236-40bb-bce7-e5114f521f03.png)